roles/matrix-dimension/tasks/setup_dimension.yml

```diff
@@ -0,0 +1,79 @@
+---
+
+#
+# Tasks related to setting up the dimension
+#
+
+- name: Ensure dimension base path exists
+  file:
+    path: "{{ matrix_dimension_base_path }}"
+    state: directory
+    mode: 0750
+    owner: "{{ matrix_user_username }}"
+    group: "{{ matrix_user_username }}"
```

Comment:
For the Dimension container to be able to access the directory (and write to it), we should use `matrix_dimension_user_gid` for `group` and a mode of `0770`.